docs/vm_builds/scripts/autounattend.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Don't we already have such an example? Do we need another? I prefer if we can just point to one.